src/styles/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "../reset.scss";
```

Comment:
The import path for 'reset.scss' should be './reset.scss' instead of '../reset.scss' if it's located in the same 'src/styles' folder.

Reply:
my reset-file locates in src, but not in styles. on the same level as index.html